util/util.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"strconv"
+	"strings"
+)
+
+// Difference returns the difference between two arrays
+func Difference(arr1, arr2 []int64) (result []int64) {
+	hash := make(map[int64]bool)
```

Comment:
What happened to the empty struct idea? why not?

Reply:
Will do